syntax = "proto2";

package dharma.common;

message VehicleParam {

  optional double front_edge_to_center = 1 [default = nan];
  optional double back_edge_to_center = 2 [default = nan];
  optional double left_edge_to_center = 3 [default = nan];
  optional double right_edge_to_center = 4 [default = nan];

  optional double length = 5 [default = nan];
  optional double width = 6 [default = nan];
  optional double height = 7 [default = nan];
}

message VehicleConfig {
  optional VehicleParam vehicle_param = 1;
}
